### ISSUE TITLE
Validation metrics not changing 

### ISSUE THREAD

**Issue by sfxnminimah** [ben-davidson-6/Gated-SCNN#10]
Hi,

I have tried using mobilenet with coco dataset of images containing person and background for my 2 classes, my train dataset contains about 81k while val contains 31k, with batch size of 50 for train and batch size of 20 for val, the training still starts with the validation accuracy being higher than the training accuracy then it drops across epochs and gets to a point of about 46% and val_mean_iou of 23% and just stops improving and get stuck while the training metrics actually improves and start from 49% and reaches as high as 86% for epoch 90, but the val metrics for accuracy and mean_iou is stuck while the loss slowly improves by like 0.00001% which is really poor.

Please any thoughts?

**Comment by benDavidson6** [comment]
If your training loss is decreasing but your validation is not it sounds like overfitting. Maybe try adding some regularisation to the mobilenet model if you haven't already. Also maybe try with a smaller training batch size of say 16.

**Comment by sfxnminimah** [comment]
Okay, thanks will try that also please how long did it take to complete 1 epoch cause mine is taking long about 7 hrs or so even tried using it on a higher gpu but no change

**Comment by sfxnminimah** [comment]
Even with reduced parameters; `Total params: 4,571,507
Trainable params: 4,528,215
Non-trainable params: 43,292
`
